lte/gateway/python/magma/mobilityd/serialize_utils.py

```diff
@@ -51,6 +51,26 @@ def _desc_state_proto_to_str(proto):
     return state
 
 
+def _desc_type_str_to_proto(type):
```

Comment:
you should either rebase these changes out of this PR in favor of #2155 or abandon that PR and just land those changes with this PR

Reply:
That is first patch, so if you review by commits in this pull request you can ignore it.